### PR TITLE
Adds benchmark support to stackrox/collector integration test workflow

### PR DESCRIPTION
**Pull request by Stringy**
Adds a new environment variable to the integration-test workflow which allows them to run the benchmarks. An alternative was to introduce an entire new workflow for the benchmarks but that felt like overkill for the outcome.

### CODE REVIEW

File: ci-operator/config/stackrox/collector/stackrox-collector-master.yaml [hunk 1]
```diff
@@ -358,11 +358,21 @@ tests:
       VM_TYPE: fedora-coreos
     workflow: stackrox-collector-integration-tests
 - as: flatcar-integration-tests
-  postsubmit: true
```
Comment (Molter73) [inline ci-operator/config/stackrox/collector/stackrox-collector-master.yaml:361]: I believe this postsubmit should be left in, right? Or were we missing the presubmit for flatcar?

Comment (Stringy) [reply ci-operator/config/stackrox/collector/stackrox-collector-master.yaml:361]: yeah there were basically two post submit jobs for flatcar (the other being the more usual `merge-flatcar-integration-tests`) so removing this make it run on PRs (admittedly only when the all-integration-tests label is provided)

**Review by Molter73 — APPROVED** [review]
/lgtm

**Pull request merged**